docs/how-to/enable-monitoring.md

```diff
@@ -199,91 +207,91 @@ The metrics exposed by etcd are detailed in the [upstream etcd documentation](ht
 The etcd charm comes with default alerts that are set up in Grafana. These alerts are based on the metrics collected by the Grafana agent. The default alerts are detailed in the [etcd documentation](https://etcd.io/docs/v3.5/op-guide/monitoring/#alerting). 
 
 They include alerts for:
-* **etcdMembersDown:**
+* ``etcdMembersDown``:
```

Comment:
Suggestion:  Correct me if I'm wrong, but I think double backticks are not needed.